File: sprint1/Java/lesson10code1.java
public static int solve(List<Integer> array) {
    int maximalSum = 0; // Сумма пустого префикса равна 0
    for (int i = 0; i < array.size(); i++) {
        int currentSum = 0;
        for (int j = 0; j <= i; j++) {
            currentSum += array.get(j);
        }
        if (maximalSum < currentSum) {
            maximalSum = currentSum;
        }
    }
    return maximalSum;
}

File: sprint1/By languages/swift/lesson10code1.swift
func solve(array: [Int]) -> Int {
    var maximalSum = 0 // Сумма пустого префикса равна 0
    for i in 0..<array.count {
        let currentSum = array[0...i].reduce(0, +)
        if maximalSum < currentSum {
            maximalSum = currentSum
        }
    }
    return maximalSum
}